verification/curator-service/api/src/model/source.ts

```diff
@@ -31,31 +31,46 @@ const fieldSchema = new mongoose.Schema({
     },
 });
 
-interface Parsing {
+interface RegexParsing {
     fields: Array<Field>;
```

Comment:
can this be [Field]? (for consistency w/ elsewhere)

Reply:
Good catch, will change this.